src/enemy.ts

```diff
@@ -23,12 +24,12 @@ export abstract class EnemyBase extends Phaser.GameObjects.Sprite {
 
         this.config = stats
         this.stats = {...stats}  // shallow copy, idk how to do deep copy
-        this.follower = { t: 0, vec: new Phaser.Math.Vector2() };
+        this.follower = { t: 0, vec: new PMath.Vector2() };
 
         this.setInteractive();
         this.on('pointerover', () => {
             if (this.scene.input.enabled)
-                (this.scene.scene.get('hudScene') as HudScene).setDescriptionEnemy(this);
+                (this.scene.scene.get<HudScene>('hudScene') as HudScene).setDescriptionEnemy(this);
```

Comment:
Do we still need `as HudScene` with the parametrized `get()`? Also in other places

Reply:
nope, fixed